dMasterServer/MasterServer.cpp

```diff
@@ -81,43 +81,18 @@ int main(int argc, char** argv) {
 	Game::logger = SetupLogger();
 	if (!Game::logger) return EXIT_FAILURE;
 
-	if (!dConfig::Exists("authconfig.ini")) {
-		LOG("Couldnt find authconfig.ini");
-		return EXIT_FAILURE;
-	}
-
-	if (!dConfig::Exists("chatconfig.ini")) {
-		LOG("Couldnt find chatconfig.ini");
-		return EXIT_FAILURE;
-	}
-
-	if (!dConfig::Exists("masterconfig.ini")) {
-		LOG("Couldnt find masterconfig.ini");
-		return EXIT_FAILURE;
-	}
-
-	if (!dConfig::Exists("sharedconfig.ini")) {
-		LOG("Couldnt find sharedconfig.ini");
-		return EXIT_FAILURE;
-	}
-
-	if (!dConfig::Exists("worldconfig.ini")) {
-		LOG("Couldnt find worldconfig.ini");
-		return EXIT_FAILURE;
-	}
+	if (!dConfig::Exists("authconfig.ini")) LOG("Couldnt find authconfig.ini.");
+	if (!dConfig::Exists("chatconfig.ini")) LOG("Couldnt find chatconfig.ini");
+	if (!dConfig::Exists("masterconfig.ini")) LOG("Couldnt find masterconfig.ini");
+	if (!dConfig::Exists("sharedconfig.ini")) LOG("Couldnt find sharedconfig.ini");
+	if (!dConfig::Exists("worldconfig.ini")) LOG("Couldnt find worldconfig.ini");
```

Comment:
Instead of removing checks like this please instead consider checking if there is an env var set which allows the bypass of the check inatead

Reply:
yep, already reveted these chagnes and am going to consider these changes in a different PR